wpilibc/src/main/native/include/frc/AddressableLED.h

```diff
@@ -0,0 +1,140 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <memory>
+
+#include <hal/AddressableLEDTypes.h>
+#include <hal/Types.h>
+#include <units/units.h>
+#include <wpi/ArrayRef.h>
+
+#include "frc/ErrorBase.h"
+
+/**
+ * A class for driving addressable LEDs, such as WS2812s and NeoPixels.
+ */
```

Comment:
This comment needs to be immediately before the class declaration.

Reply:
Hmm. This was fixed, not sure why GitHub isn't showing it.